events/schema.py

```diff
@@ -977,7 +996,10 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
 class Query:
     events = DjangoFilterConnectionField(EventNode)
     events_and_event_groups = graphene.ConnectionField(
-        EventOrEventGroupConnection, project_id=graphene.ID()
+        EventOrEventGroupConnection,
+        project_id=graphene.ID(),
+        upcoming=graphene.Boolean(),
+        upcoming_with_leeway=graphene.Boolean(),
```

Comment:
Would it be better to let the client decide the amount of leeway? If so, then the Boolean should be changed to a number.